wfe/wfe.go

```diff
@@ -388,7 +403,7 @@ func (wfe *WebFrontEndImpl) NewRegistration(
 	response http.ResponseWriter,
 	request *http.Request) {
 
-	body, key, prob := wfe.verifyPOST(ctx, logEvent, request)
+	body, key, prob := wfe.verifyPOST(ctx, logEvent, request, wfe.extractJWK)
```

Comment:
Can you add a brief comment here describing why this is the only place to use `extractJWK` vs `lookupJWK`?